File: lv1/이상한 문자.py
def solution(s):
    answer = ''
    for n in s.split(" "):
      answer+= " "
      for i in enumerate(n):
        if i[0] % 2 == 1:
          answer += i[1].lower()
        else:
          answer += i[1].upper()
  
    return answer[1:]

File: lv1/이상한 문자.js
function solution(s) {
  const Sarr = s.split(" ").map(e => [...e].map((e, i) => {
    if ((i + 1) % 2 === 0) return e.toLowerCase();
    if ((i + 1) % 2 === 1) return e.toUpperCase()
  }).join("")).join(" ");
  return Sarr;
}